.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+omit = */migrations/*
+       */fielded_batch_ingest.py
+       */csv_to_pickle.py
+       */dynamic_form.py
+       */batch_valuesets.py
```

Comment:
Three of them were command-line utility scripts, and the other is no longer used. So since they aren't really a part of the django app, I decided they shouldn't be affecting the coverage reports. They will either be deleted or factored into the app (with tests) with the 1.0.0 milestone.